README.md

```diff
@@ -116,12 +116,15 @@ the following keys:
 
 - `platformBlacklist`
 
-  This is a list of platform names, optionally followed by the
-  version, where this test is invalid . For example, `["fedora"]`
-  means this test should be skipped on all Fedora platforms.
-  `["rhel7"]` means this test should be skipped on RHEL 7, but not on
-  RHEL 8, or another RHEL version.
+  This is a list of runtime-ids or platform names (optionally followed
+  by the version) where this test is invalid.
 
+  Examples:
+
+  - `["linux-arm64"]`: skip this test on `arm64` (aka `aarch64`)
+  - `["fedora"]`: skip this test on all Fedora platforms
+  - `["rhel.7"]` or `["rhel7"]`: skip this test on RHEL 7, but not on RHEL 8, or another RHEL version
+  - `["rhel.8-arm64"]`: skip this test on RHEL 8 on arm64
```

Comment:
Note: `rhel8-arm64` (without the dot) is not supported.

Reply:
I am not sure actually. Both having it and not having it make sense.

Reply:
Long term, I want to get rid of the `rhel8`-style elements and just use the RID-style everywhere. Do you see an problems with this approach? Is there any platform that we wont be able to express using just RIDs?

Reply:
Nope, sounds good. (Assuming it's reflected in the test repositories)